vm/src/interpreter.cpp

```diff
@@ -389,8 +441,13 @@ bool run_frame(runtime_state &state, stack_frame &frame, CallStack &call_stack,
 
             // Replace the current process with a new process image
             if (execvp(argv[0].get(), reinterpret_cast<char *const *>(argv.data())) == -1) {
-                perror("execvp");
-                _exit(MOSHELL_COMMAND_NOT_RUNNABLE);
+                std::stringstream ss;
+                ss << argv[0].get();
+                for (int i = 1; i < frame_size; i++) {
+                    ss << " " << std::string(argv[i].get());
+                }
+                panic("Unable to execute command \"" + ss.str() + "\": " + std::string(strerror(errno)), call_stack);
```

Comment:
```suggestion
                std::string command = argv[0].get();
                panic("Unable to execute command \"" + command + "\": " + std::string(strerror(errno)), call_stack);
```
An `ostream` is not particularly needed here, I am not very convinced that the full command should be printed.